kura/org.eclipse.kura.container.provider/OSGI-INF/metatype/org.eclipse.kura.container.provider.ContainerInstance.xml

```diff
@@ -60,10 +46,24 @@
             description="Password for container registry. Required only for authenticated registries" 
             type="Password"
             cardinality="0" required="false" default="" />
+
+        <AD id="container.signature.trust.anchor" name="Trust anchor"
+            description="Trust anchor used to verify the container image signature.|TextArea" type="String"
+            cardinality="1" required="false" default="" />
+
+        <AD id="container.signature.verify.transparency.log" name="Verify in transparency log"
+            description="Sets the transparency log verification, to be used when a container image signature has been uploaded to the transparency log." type="Boolean"
+            cardinality="1" required="false" default="true" />
+        
+        <AD id="container.signature.enforcement.digest"
+            name="Container Image Enforcement Digest"
+            description="Digest of the container image allowed to run on the device, if the Container Enforcement Monitor is enabled. If not provided, it will be computed by the Container Signature Verification service."
+            type="String" cardinality="1" required="false"
+            default="" />
         
         <AD id="container.image.download.retries" 
             name="Image Download Retries" 
-            description="The number of retries to pull the container image. Set to 0 for unlimited retries"
+            description="Specifies the number tentatives to pull the container image that the framework performes. Set to 0 for unlimited retries. Default: 5"
```

Comment:
"Tentatives" is not a common way to express this. I think "retries" is correct here.

```suggestion
            description="Specifies the number retries the framework performs when attempting to pull the container image. Set to 0 for unlimited retries. Default: 5"
```